new api version implemented

dmytrolutsyk opened the pull request.

dmytrolutsyk committed:
new api version implemented

dmytrolutsyk committed:
add update category method

dmytrolutsyk committed:
update prilary category miss secondary category

dmytrolutsyk committed:
add app version and info localizations update and this commit contains to app categories update

fehmitoumi committed:
added missing object definition for AppStoreVersion relationships
modified add_build_to_beta_group because it does not return a BetaGroup object
modified modify and create app_store_version functions

Tuni commented on appstoreconnect/api.py:
```diff
@@ -1,17 +1,27 @@
 	PATCH = 3
+	DELETE = 4
 
 class AppStoreState(Enum):
```
Je me demande si cette classe a sa place ici. Ca a aucune utilité à la lib il me semble, c'est utilisé pour notre logique à nous. T'en penses quoi ?

dmytrolutsyk replied:
Je suppose que dans notre usage, la suppression d'élément ne doit se faire à aucun moment sans l'intervention humaine. C'est pas forcément la bonne idée d'automatiser la suppression. On en n'a pas besoin. Mais dans la lib la méthode delete ne représente aucun danger, on l'appelle nulle part .

fehmitoumi committed:
deleted AppStoreState class

dmytrolutsyk committed:
Merge pull request #11 from Tuni/DevNewApiVersion_cleanup

deleted AppStoreState class

fehmitoumi committed:
Merge branch 'DevNewApiVersion' into appInfoLocalization

fehmitoumi committed:
fixed typo

fehmitoumi committed:
added again modify_app_store_version_localization ( removed accidentaly )
added copyroth parameters form create and modify app store version functions

dmytrolutsyk committed:
Update api.py

Change localisation of functions

dmytrolutsyk committed:
change parameters of modify_app_info

dmytrolutsyk committed:
change list_app_store_info_localizations and add doc ref

dmytrolutsyk committed:
- change modify_app_store_info_localization and modify_app_store_version_localization parameters
- add reference for list_app_store_version_localizations and list_app_store_info_localizations

dmytrolutsyk committed:
Merge pull request #12 from Tuni/refactorAppInfoLocalization

Refactor app info localization

dmytrolutsyk committed:
Merge pull request #9 from Tuni/appInfoLocalization

App info localization

dmytrolutsyk committed:
add AgeRatingDeclarations class and methods to use read and modify agerating

dmytrolutsyk committed:
Update resources.py

fix coma bug in AppStoreVersionLocalization class attributes list

dmytrolutsyk committed:
factorize read_age_rating_declarations_info, add good reference, refactor the name of function to lowercase string

Tuni committed:
Merge pull request #14 from Tuni/ageRatingrefacto

Age ratingrefacto
